zebra-rpc/src/methods.rs

```diff
@@ -1851,12 +1845,18 @@ where
                 .await
                 .map_misc_error()?
             {
-                zebra_state::ReadResponse::SaplingTree(tree) => tree.map(|t| t.to_rpc_bytes()),
+                zebra_state::ReadResponse::SaplingTree(tree) => tree.map(|t| {
+                    let mut root = Into::<[u8; 32]>::into(t.root()).to_vec();
+                    root.reverse();
+                    (t.to_rpc_bytes(), root)
```

Comment:
Subjective/Nitpick/Optional: 
- It may be a little more readable without the trait qualifier
- Do we want to move the reversal to a `bytes_in_display_order()` method?

```suggestion
                    let mut root: [u8; 32] = t.root().into();
                    root.reverse();
                    (t.to_rpc_bytes(), root.to_vec())
```

Reply:
Good idea, `bytes_in_display_order()` sounds good (we do have an inconsistency that we also have `to_rpc_bytes()` but we can sort that out later). Done in 99e6bf0c3